becca-new-app/src/TodoList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+const todoList = [
+    { id: 1, title: 'Read over assignments'},
+    { id: 2, title: 'Exercise for 1 hour'},
+    { id: 3, title:  'Make dinner'}
+];
+const TodoList = () => {
+    return (
```

Comment:
Please indent your code. Children are always indented from its parent. Children on the same level have the same indentation.